### PR TITLE
Updates schema for Google Cloud Marketplace

### PR DESCRIPTION
**Pull request by meik99**
# Description

Updates the schema for the Google Cloud Marketplace to reflect the latest changes to the values.yaml and Helm charts.

## How can this be tested?

* Install `mpdev`, see [google documentation for more information](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/tool-prerequisites.md)
* Create an empty GKE cluster
* Apply Googles Application CRD, see [google documentation for more information](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/tool-prerequisites.md)
* Run `hack/gcr/deployer-image.sh' to build and push a new deployer image containing the helm charts
* Run `hack/gcr/deploy.sh` to deploy the deployer image
* In the Google Cloud Console, go to Kubernetes Clusters / Applications, select your cluster and check the deployment was successful

## Checklist
~~- [ ] Unit tests have been updated/added~~
- [x] PR is labeled accordingly

### CODE REVIEW

File: config/helm/chart/default/Chart.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ description: The Dynatrace Operator Helm chart for Kubernetes and OpenShift
 icon: https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_512x512px.png
 home: https://www.dynatrace.com/
 type: application
-version: 0.4.2
-appVersion: 0.4.2
+version: 0.6.0
```
Comment (0sewa0) [inline config/helm/chart/default/Chart.yaml:21]: shouldn't this be `snapshot` for master?

File: config/helm/schema.yaml [hunk 2]
```diff
@@ -125,23 +191,151 @@ properties:
...
+
+  installCRD:
+    type: boolean
+    title: Install DynaKube CRD
+    description: |
+      Tells the Helm chart of the deployer to install the custom resource definition (CRD) of the DynaKube API.
+    default: true
     enum:
-    - 'yes'
+      - true
```
Comment (0sewa0) [inline config/helm/schema.yaml:329]: does the enum needed ?
with this user can't set `installCRD` to `false`

**Comment by 0sewa0** [comment]
Updating the https://github.com/Dynatrace/dynatrace-operator/blob/master/config/helm/README.md with the info in this PR's description would be super awesome

File: config/helm/chart/default/Chart.yaml (hunk 1 above)
Comment (meik99) [reply config/helm/chart/default/Chart.yaml:21]: snapshot does not work, has to be semver

File: config/helm/schema.yaml (hunk 2 above)
Comment (meik99) [reply config/helm/schema.yaml:329]: The schema is used for inital installation of the operator. Therefore it makes no sense to not install it

Comment (0sewa0) [reply config/helm/schema.yaml:329]: @chrismuellner should we give the option ? (I think we should)

File: config/helm/chart/default/Chart.yaml (hunk 1 above)
Comment (0sewa0) [reply config/helm/chart/default/Chart.yaml:21]: 😢

**Review by 0sewa0 — APPROVED** [review]

**Pull request merged**